CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Fixed
 
+- `BitmovinAnalyticsConfig` is serializable now. `Context` is now passed seperately where needed.
```

Comment:
I don't think this should go into the fixed category as it wasn't really a bug right?

You can also add a `# Internal` category for changes that don't go into the public changelog on the website